ui/src/app/controller/pipes/date-to-utc.pipe.ts

```diff
@@ -7,13 +7,14 @@ import moment from "moment";
 })
 export class DateToUtcPipe implements PipeTransform {
 
-  transform(localDate: Date, localTime: string, port: Port): string {
+  transform(localDate: Date, localTime: string, port: Port): Date {
     let year = localDate.getFullYear();
     let month = String((localDate.getMonth() + 1)).padStart(2, '0');
     let day = String(localDate.getDate()).padStart(2, '0');
     let [hour, minute] = localTime.split(':');
     let second = String(localDate.getSeconds()).padStart(2, '0');
     // For whatever reason, this only works when passing date as string instead of a Date object
-    return moment.tz(`${year}-${month}-${day} ${hour}:${minute}:${second}`, port.timezone).toISOString();
+    let newDate = moment.tz(`${year}-${month}-${day} ${hour}:${minute}:${second}`, port.timezone).toISOString()
+    return new Date(newDate);
```

Comment:
@nt-as when we are retrieving results from the database how does it handle timezone conversion? Do we manually convert according to the port? If the database converts to and from the timezone of the Date object there might be an error because Data objects only support local timezones.